### PR TITLE
Update navigation.lang, add footer.lang

### PR DESCRIPTION
**Pull request by craigcook**
Compare to bedrock PR https://github.com/mozilla/bedrock/pull/7755 (I think I got everything...)

navigation.lang is a globally shared file but these are only new strings, not changing any existing ones in the existing nav bar. The new Firefox nav will be conditional to English until the new strings are complete.

footer.lang is new and will eventually be a globally shared file as well, and both versions of the footer will be conditional to English until all strings are complete.

Demo: 
Mozilla nav and footer: https://bedrock-demo-shobson.oregon-b.moz.works  
Firefox nav and footer: https://bedrock-demo-shobson.oregon-b.moz.works/firefox/

### CODE REVIEW

File: en-US/navigation.lang [hunk 1]
```diff
@@ -303,3 +309,183 @@ Get a Firefox Account
...
+
+;Firefox for Enterprise
+Firefox for Enterprise
+
+
+;Get unmatched data protection on the release cadence that suits your organization.
+Get unmatched data protection on the release cadence that suits your organization.
+
+
+;Firefox Reality
```
Comment (peiying2) [inline en-US/navigation.lang]: If this is the product name, which means it is left in English. Then we can remove this string.

**Comment by peiying2** [comment]
@craigcook lots of update since Friday. Let me know if this is final and ready for an official review.

**Comment by craigcook** [comment]
Yes I think it's final.

File: en-US/navigation.lang [hunk 2]
```diff
@@ -303,3 +310,175 @@ Get a Firefox Account
...
+;Close Browsers menu
+Close Browsers menu
+
+
+# please do not translate the product name "Firefox Browser"
+;Firefox Browser for Desktop
+Firefox Browser for Desktop
+
+
+;Get the browser that respects your privacy automatically. On Windows, MacOS or Linux.
```
Comment (peiying2) [inline en-US/navigation.lang]: `MacOS` should be `macOS`

File: en-US/navigation.lang [hunk 3]
```diff
@@ -303,3 +310,175 @@ Get a Firefox Account
...
+# please do not translate the product name "Firefox Browser"
+;Firefox Browser for Enterprise
+Firefox Browser for Enterprise
+
+
+;Get unmatched data protection on the release cadence that suits your organization.
+Get unmatched data protection on the release cadence that suits your organization.
+
+
+;Strap on your goggles and step into the immersive web with Firefox’s VR browser.
```
Comment (peiying2) [inline en-US/navigation.lang]: Firefox as a brand must be left unchanged. This format "Firefox's" xyz will force some languages to declense the brand so it looks like Firefoxa or Firefoksa to be grammatically correct to name a few. This is for languages using Latin scripts. Can we change the wording to make it easier for localizer, like `VR browser of Firefox`. For other non-latin scripts, some face equally challenging situation.

Comment (craigcook) [reply en-US/navigation.lang]: We've asked for a different version of this blurb, just waiting for that to be final.

Comment (craigcook) [reply en-US/navigation.lang]: Updated!

**Comment by craigcook** [comment]
OK, actually final now.

**Comment by craigcook** [comment]
Sources updated in https://github.com/mozilla-l10n/langchecker/pull/928

**Pull request merged**